Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -400,24 +404,21 @@ extension HTTPClient {
     /// Response execution context. Will be created by the library and could be used for obtaining
     /// `EventLoopFuture<Response>` of the execution or cancellation of the execution.
     public final class Task<Response> {
-        /// `EventLoop` used to execute and process this request.
+        @available(*, deprecated, renamed: "eventLoop")
```

Comment:
same here

Reply:
filed https://github.com/swift-server/async-http-client/issues/108 to remove them just before

Reply:
@artemredkin happy with #108 ?

Reply:
sure, thanks!